__tests__/data/attribution_input.json

```diff
@@ -23,37 +23,16 @@
       "anonymousId": "ac7722c2-ccb6-4ae2-baf6-1effe861f4cd",
       "channel": "web",
       "context": {
-        "app": {
```

Comment:
Rather than replacing the whole context object with dummy values, it'd be better if we can make it look like a real event